application/gqlgen/schema.graphql

```diff
@@ -65,14 +65,27 @@ type User {
     organizations: [Organization!]!
     posts(role: PostRole!): [Post!]!
     photoURL: String!
+    preferences: [UserPreference!]
     location: Location
     unreadMessageCount: Int!
 }
 
+type UserPreference {
+    id: ID!
+    key: String!
+    value: String!
+}
+
 input UpdateUserInput {
     id: ID
     photoID: String
     location: LocationInput
+    preferences: [UpdateUserPreferenceInput!]
```

Comment:
```suggestion
    preferences: [UpdateUserPreferenceInput!]!
```
I'm not sure of this one, since we've never done a migration with a list.

Reply:
A migration? :-)

Reply:
lol, mutation